policy/modules/contrib/networkmanager.fc

```diff
@@ -26,7 +26,7 @@
 
 /usr/sbin/NetworkManager	--	gen_context(system_u:object_r:NetworkManager_exec_t,s0)
 /usr/sbin/wpa_supplicant	--	gen_context(system_u:object_r:NetworkManager_exec_t,s0)
-/usr/sbin/NetworkManagerDispatcher --	gen_context(system_u:object_r:NetworkManager_exec_t,s0)
+/usr/sbin/NetworkManagerDispatcher --	gen_context(system_u:object_r:NetworkManager_dispatcher_exec_t,s0)
```

Comment:
I can see only /usr/libexec/nm-dispatcher on my system, but supposedly we are leaving this as a legacy.